src/ci.rs

```diff
@@ -1,21 +1,29 @@
 use crate::schema::Team;
 use anyhow::Context;
+use std::collections::BTreeSet;
 use std::path::{Path, PathBuf};
 
+struct PrivilegedUsers {
+    /// Users that have all the privileges.
+    admins: Vec<String>,
+    /// Users with `write` access that can only modify data files.
```

Comment:
can you also list people with maintain access?

Reply:
Done.